cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -61,8 +53,9 @@ CDAP:
 #. Use the Cloudera Manager UI to download, distribute, and activate the target
    CDAP parcel version on all cluster hosts.
 
-#. Before starting services, run the *CDAP Upgrade Tool* to upgrade CDAP. From the CDAP Service 
-   page, select "Run CDAP Upgrade" from the Actions menu.
+#. Before starting services, run the *CDAP Upgrade Tool* to upgrade CDAP. From the CDAP Service
+   page, select "Run CDAP Upgrade" from the Actions menu. Note that authorization is disabled in
+   the UpgradeTool, so that the cdap user could upgrade all user's data.
```

Comment:
It can be more than just datasets though, so I just mentioned data.
It can be streams, inter-flowlet queues, application metadata, etc.